### PR TITLE
UI: Moves fetchdata() to the created()

### PR DESCRIPTION
**Pull request by utchoang**
### Description
This PR to move fetchData() to created() insteadof mount()

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Enhancement (improves an existing feature and functionality)
- [x] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [x] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### CODE REVIEW

**Comment by utchoang** [comment]
@blueorangutan ui

**Comment by shwstppr** [comment]
@utchoang will this offer any performance improvement? I hope we are not accessing/manipulating DOM elements in any of the fetchData methods

**Comment by utchoang** [comment]
@shwstppr Yes. In https://github.com/apache/cloudstack-primate/issues/719 then move fetchData to created from mounted to reduce wait time for API calls since the data has been created but the page has not yet been rendered

**Comment by rhtyd** [comment]
@utchoang what are the pros and cons of using one vs the other?

**Comment by utchoang** [comment]
@rhtyd When the component is initialized, fetching data from the backend at `created()` will be done without having to wait for the DOM to be mounted or rendered as if it were called on `mounted()`.

File: ui/src/views/infra/network/EditTrafficLabel.vue [hunk 1]
```diff
@@ -143,8 +143,7 @@ export default {
       })
     }
   },
-  inject: ['parentFetchData'],
```
Comment (davidjumani) [inline ui/src/views/infra/network/EditTrafficLabel.vue:146]: @utchoang Removed by accident ?

File: ui/src/views/offering/ImportBackupOffering.vue [hunk 2]
```diff
@@ -128,10 +128,9 @@ export default {
       this.apiParams[param.name] = param
     })
   },
-  mounted () {
+  created () {
     this.fetchData()
   },
-  inject: ['parentFetchData'],
```
Comment (davidjumani) [inline ui/src/views/offering/ImportBackupOffering.vue:134]: Here too ?

File: ui/src/views/infra/network/EditTrafficLabel.vue (hunk 1 above)
Comment (utchoang) [reply ui/src/views/infra/network/EditTrafficLabel.vue:146]: @davidjumani No, delete as it is not used in this component.

Comment (davidjumani) [reply ui/src/views/infra/network/EditTrafficLabel.vue:146]: Line 198 ?

Comment (utchoang) [reply ui/src/views/infra/network/EditTrafficLabel.vue:146]: Ah. Maybe I am confused with the PR async job. I'll fix it now.

**Comment by davidjumani** [comment]
@blueorangutan ui

**Review by davidjumani — APPROVED** [review]
LGTM. Did a quick round of testing
Forms appear to load a bit sooner

**Comment by rhtyd** [comment]
@utchoang can you fix the conflict against latest 4.15 branch

**Comment by rhtyd** [comment]
@utchoang can you fix the conflict?

**Comment by rhtyd** [comment]
@blueorangutan ui

**Review by rhtyd — APPROVED** [review]
LGTM proxing David's testing/review

**Pull request merged**